titiler/stacapi/stac_reader.py

```diff
@@ -12,9 +14,10 @@
 from rio_tiler.io import BaseReader, Reader, stac
 from rio_tiler.types import AssetInfo
 
-from titiler.stacapi.settings import STACSettings
+from titiler.stacapi.settings import STACAPISettings, STACSettings
 
 stac_config = STACSettings()
+stac_api_config = STACAPISettings()
```

Comment:
This setting is not used?

Reply:
thanks for noticing this, I had misconfigured the pre-commit linter so missed it!